### DISCUSSION TITLE
What are you using for docs?

### DISCUSSION THREAD

**Discussion by iqbal125** [mantinedev/mantine#12] [Q&A] [answered]
Random question, but what did you use to build the docs it looks really good? Or is it a custom job?

**Comment by rtivital** [comment, marked as answer]
It’s Gatsby with mdx, docs are made from scratch, you can find source code in docs folder

**Reply by steph4nc** [reply]
I see many of the components for the docs are custom made on top of mantine. Would it be possible to abstract some of those components into reusable mantine components. Like the top nav menu, the side navbar, etc. Then the docs are more pure mantine, without many custom components.

**Reply by rtivital** [reply]
These components are designed for mantine docs, it'll take lots of work to make them generic and right now I do not have plans to do so (until 2.0.0 is out at least), meanwhile you can copy and paste source code from mantine docs, website has MIT license too.

**Comment by iqbal125** [comment]
@rtivital, Good job my man, last time I looked at this library it was at 186 stars and its already over 500 in just a few days.
